recipes/uwsgi/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: uwsgi
```

Comment:
Does the conda forge project encourage letter casing?  I think Ilan and repo.continuum.io explicitly discourage it and lowercase everything.  pyyaml instead of PyYAML for example.